.gitignore

```diff
@@ -31,3 +31,6 @@ pip-log.txt
 
 newsdiffs.db
 database_settings.py
+
+/*.venv/
+/.idea/
```

Comment:
No newline at end of file!

Reply:
Is that causing an error?  Or just a style catch?